### ISSUE TITLE
Better (worse) multithreading request

### ISSUE THREAD

**Issue by WilliamAnimate** [Pissandshittium/pissandshittium#43]
When the browser launches, it should start a thread running useless calculations, and every 15 minutes it should start more threads that do the same.

**Comment by Setsugennoao** [comment]
Why useless calculations when they could be bitcoin miners and use your GPU as well!

**Comment by WilliamAnimate** [comment]
Good idea, but the bitcoin shouldn't be sent to the owner of the machine, but to the creator of `Pissandshittium`.

**Comment by polypoyo** [comment]
Better: mined coins are sent to `000000000000000000000000000000000000000000`

**Comment by Setsugennoao** [comment]
> Good idea, but the bitcoin shouldn't be sent to the owner of the machine, but to the creator of `Pissandshittium`.

Yeah I took that for granted

> Better: mined coins are sent to `000000000000000000000000000000000000000000`

This is a good idea, or maybe the wallet address could be randomized?
